lldb/include/lldb/API/SBFrame.h

```diff
@@ -233,6 +233,10 @@ class LLDB_API SBFrame {
 
   void SetFrameSP(const lldb::StackFrameSP &lldb_object_sp);
 
+  /// Return an SBValue with an error message warning that the process is not
+  /// currently stopped.
```

Comment:
Nit: The "error message warning" is somewhat confusing. How about:
```suggestion
  /// Return an SBValue containing an error message that warns the process is not currently stopped.
```